testnet/tools/nns-tools/list-new-commits.sh

```diff
@@ -78,10 +78,15 @@ list_new_canister_commits() {
         print_cyan "${HEADING}"
     fi
 
-    # Print commits.
-    if [[ "${INTERESTING_COMMITS}" != "" ]]; then
-        echo "${INTERESTING_COMMITS}"
-    fi
+    while IFS= read -r COMMIT_MSG; do
+        if [[ $(echo "$COMMIT_MSG" | grep -v -E ' .{10} (chore|refactor|test)\b' | wc -l) -gt 0 ]];
+        then
+            print_blue "${COMMIT_MSG}"
```

Comment:
```suggestion
        if echo "$COMMIT_MSG" | grep -E ' .{10} (chore|refactor|test)\b'; then
            print_light_grey ...
```

Reply:
Doesn't work exactly right, going to skip suggestion:
![image](https://github.com/user-attachments/assets/6b12c13a-4676-48d1-b096-1d2d763a9043)